test/TestServerProjects/paging/Generated/Operations/PagingOperations.cs

```diff
@@ -2353,12 +2353,12 @@ public AsyncPageable<Product> GetNullNextLinkNamePagesPageableAsync(Cancellation
             async Task<Page<Product>> FirstPageFunc(int? pageSizeHint)
             {
                 var response = await GetNullNextLinkNamePagesAsync(cancellationToken).ConfigureAwait(false);
-                return Page.FromValues(response.Value.Values, null, response.GetRawResponse());
+                return Page.FromValues(response.Value.Values, response.Value.NextLink, response.GetRawResponse());
```

Comment:
As mentioned above, it was null because there was a not a value provided. We still need to support this.

Reply:
Super strange but ok :)